experimental/builder/include/ck_tile/builder/testing/extent.hpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Advanced Micro Devices, Inc. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+namespace ck_tile::builder::test {
+
+/// This structure describes a 1-, 2-, or 3-D extent. Its used to
+/// communicate 1-, 2- or 3-D sizes and strides of tensors.
```

Comment:
I don't see information about strides - or you assume that tensors are always packed?

Reply:
Thats correct, though thats unrelated to this type (see its use in conv_fwd.hpp). The functionality from CK that I'm using to construct tensor descriptors does not actually support non-packed tensors right now, and rather than changing that in this PR I figured to do it as part of follow-up work. I've already created an internal ticket to keep track of that.